lib/zendesk_apps_support/package.rb

```diff
@@ -121,6 +121,31 @@ def translation_files
       files.select { |f| f =~ %r{^translations/} }
     end
 
+    def obj(options)
+      begin
+        app_id = options.fetch(:app_id)
+        asset_url_prefix = options.fetch(:assets_dir)
+        name = options.fetch(:app_name)
+      rescue KeyError => e
+        raise ArgumentError, e.message
+      end
+
+      app_class_name = "app-#{app_id}"
+
+      {
+        app_class_name: app_class_name,
+        app_class_properties: manifest.app_class_properties,
+        asset_url_prefix: asset_url_prefix,
+        author: manifest.author,
+        framework_version: manifest.framework_version,
+        id: app_id,
+        location_icons: location_icons,
```

Comment:
The function here https://github.com/zendesk/zendesk_apps_support/pull/266/files#diff-f9dc5da4f07a7cd59a608b6da769f0ecR336-R349 should also be moved to ZAF and this option instead replaced with `location_options: manifest.location_options`